src/NHibernate.Test/Extralazy/ExtraLazyFixture.cs

```diff
@@ -106,6 +107,51 @@ public void OrphanDelete()
 			}
 		}
 
+		[Test]
+		public void OrphanEnumerableDelete()
+		{
+			User gavin = null;
+			Widget widget = null;
+			Widget widget2 = null;
+
+			using (var s = OpenSession())
+			using (var t = s.BeginTransaction())
+			{
+				gavin = new User("gavin", "secret");
+				gavin.AddWidget("widg1", "blah blah blah");
+				gavin.AddWidget("widg2", "blah blah blah blah");
+				s.Persist(gavin);
+				t.Commit();
+			}
+
+			using (var s = OpenSession())
+			using (var t = s.BeginTransaction())
+			{
+				gavin = s.Get<User>("gavin");
+				widget = s.Get<Widget>("widg1");
+				//Assert.IsTrue(gavin.Widgets.Any(w => w == widget));
```

Comment:
```suggestion
```

Reply:
Un-commenting this line makes the test pass, demonstrating the issue.

Reply:
I understand, could you then put this into the comment?